### ISSUE TITLE
[BUG] Author's Note window doesn't appear when button is pressed

### ISSUE THREAD

**Issue by seismic-toss** [SillyTavern/SillyTavern#2604]
### Environment

🐋 Docker

### System

Docker 24.0.9; Firefox 128.0.2

### Version

1.12.4

### Desktop Information

Docker image branch: Staging (963b849f032574c1bc250ec689cc162658c5f48e87794fe65136bb420568fdb7)

### Describe the problem

Pressing the Author's Note button in the chat controls doesn't cause the Author's note panel to show up. It worked as expected previously, with no change to configuration between. 

I'm not all that knowledgeable about html and css, but I went digging through the inspector to find the div. It *is* modifying its display property and such, but is setting an inset that appears to send it outside of the viewbox. Disabling that property does make it appear, albeit using its maximum 90% width for some reason; it did not do that when it was working properly before. Notably, it will appear when the viewbox is small enough that it snaps to full width.
But, obviously, flipping html properties isn't intended use. And I'm not knowledgeable enough to dig through and find the cause.

### Additional info

[console-export-2024-8-2_16-16-11.txt](https://github.com/user-attachments/files/16476058/console-export-2024-8-2_16-16-11.txt)
![Screenshot from 2024-08-02 15-36-18](https://github.com/user-attachments/assets/e05b8a6d-6c2d-42a8-b7c8-546e4971bcab)
![Screenshot from 2024-08-02 15-36-22](https://github.com/user-attachments/assets/cea6e2ea-a2a5-47a9-8cfd-c1cbcc92f1e9)
![Screenshot from 2024-08-02 15-36-29](https://github.com/user-attachments/assets/bfe66cc7-930c-44e8-bf01-2f71c8da5f66)

### Please tick the boxes

- [X] I have explained the issue clearly, and I included all relevant info
- [X] I have checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] I have checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Wolfsblvt** [comment]
This one comes from Moving UI scaling.
Can you try to reset your Moving UI, or disable it for now, just for testing?

You can also confirm this comes from this by checking the dev console, which should log something like
```
scaling floatingPrompt by 00x00 to 000x000
```

Do you actually use ST on multiple systems? Multiple different screens/resolutions? And use Moving UI?

**Comment by seismic-toss** [comment]
> Can you try to reset your Moving UI, or disable it for now, just for testing?

Disabling Moving UI was actually the first step I took when it stopped working. I had it off for the last couple days (through multiple container restarts), hoping the panel would show up at some point. I didn't actually realize that the 'MUI' in the settings was referring to Moving UI until looking for the reset button you mentioned... sure enough, that seems to fix it: I can close and open the AN panel without the inspector now. Maybe that MUI heading could be a bit more obvious.  
I don't use it on multiple systems; my desktop IP is the only one in the whitelist. My only thought about how *I* might have displaced the panel outside of the viewport is maybe when dragging the window between monitors with different resolutions?

**Comment by Wolfsblvt** [comment]
Yeah, that might be the reason and what has happened. Once MovingUI *saved* it outside of the viewport, it'll always get rendered there, even before you see the ST window itself. Moving UI is automatically being applied.

Might have to look into making this a bit more obvious, or fixing when it is fully outside of the viewport.

**Comment by seismic-toss** [comment]
I appreciate your help finding this quick fix, and respect your contribution to the project. Thank you.
